src/components/editor/monaco.ts

```diff
@@ -98,10 +98,43 @@ function addEditorKeyboardShortcuts(
         contextMenuGroupId: 'starboard',
         contextMenuOrder: 1,
         run: (_ed) => emit({
-            id: cellId, type: "RUN_CELL", focusNextCell: true, insertNewCell: false
+            id: cellId, type: "RUN_CELL", focus: "next"
         })
     });
 
+    editor.addAction({
+        id: 'run-cell-and-next',
```

Comment:
rename (also below) to something like "Focus next cell"

Reply:
Okay, fixed!